[unpleasant] Update editorial suggestions

SylviaZhaooo opened the pull request:
Resolve most comments in #445.

SylviaZhaooo committed:
[unpleasant] Update editorial suggestions

HumphreyYang committed:
update links between lectures

mmcky commented on lectures/unpleasant.md:
```diff
@@ -11,14 +11,14 @@ kernelspec:
 This lecture builds on concepts and issues introduced in our  lecture on **Money Supplies and Price Levels**.
 
-That lecture describes stationary equilibria that reveal a **Laffer curve** in the inflation tax rate and the associated  stationary rate of return 
+That lecture describes stationary equilibria that reveal a [**Laffer curve**](https://en.wikipedia.org/wiki/Laffer_curve) in the inflation tax rate and the associated  stationary rate of return 
```
Could we change `Laffer curve` from bold to italic.

mmcky commented on lectures/unpleasant.md:
```diff
@@ -152,15 +154,15 @@ running monetary and fiscal policies.
 Here, by **fiscal policy** we mean the collection of actions that determine a sequence of net-of-interest government deficits $\{g_t\}_{t=0}^\infty$ that must be financed by issuing to the public  either money or interest bearing bonds.
 
-By **monetary policy** or **debt-management polcy**, we  mean the collection of actions that determine how the government divides its  portolio of debts to the public  between interest-bearing parts (government bonds) and non-interest-bearing parts (money).
+By **monetary policy** or **debt-management policy**, we  mean the collection of actions that determine how the government divides its  portolio of debts to the public  between interest-bearing parts (government bonds) and non-interest-bearing parts (money).
```
can we change these from bold to italic.

mmcky commented on lectures/unpleasant.md:
```diff
@@ -252,9 +254,9 @@ Python coder.
 To compute an equilibrium, we deploy the following algorithm.
 
-Given **parameters** include $g, \check m_0, \check B_{-1}, \widetilde R >1, T $
+Given **parameters** include $g, \check m_0, \check B_{-1}, \widetilde R >1, T $.
```
can we change this from bold to italic

mmcky commented on lectures/unpleasant.md:
```diff
@@ -501,11 +494,13 @@ mystnb:
 * The lower is the post-open-market-operation money supply at time $0$, lower is the price level at time $0$.
 
-* An open  market operation that reduces the post-open-market-operation money supply at time $0$ also **lowers** the rate of return on money $R_u$ at times $t \geq T$ because it brings  a higher gross-of-interest government deficit that must be financed by printing money (i.e., levying an inflation tax) at time $t \geq T$.
+* An open  market operation that reduces the post-open-market-operation money supply at time $0$ also *lowers* the rate of return on money $R_u$ at times $t \geq T$ because it brings  a higher gross-of-interest government deficit that must be financed by printing money (i.e., levying an inflation tax) at time $t \geq T$.
```
I would suggest we change `post-open-market-operation` to `post open market operation`

similar comment for `gross-of-interest`

mmcky reviewed (changes requested):
@SylviaZhaooo Thanks for your work on this. Just included some suggestions for update.

SylviaZhaooo committed:
Update unpleasant.md

mmcky commented:
thanks @SylviaZhaooo if you have addressed the comments above. Would you mind clicking on the "Resolved comment". Thanks 👍

SylviaZhaooo commented:
> thanks @SylviaZhaooo if you have addressed the comments above. Would you mind clicking on the "Resolved comment". Thanks 👍
Thanks for reminding me. I have clicked on all "Resolved comments".

mmcky reviewed (approved):
thanks @SylviaZhaooo. 

@jstac this is ready for final review.

jstac commented on lectures/unpleasant.md:
```diff
@@ -91,8 +93,8 @@ $$
 
-Notice that $\check m_0$ is a **nominal** quantity, being measured in dollar, while
-$\widetilde R \check B_{-1}$ is a **real** quantity, being measured in time $0$ goods.
+Notice that $\check m_0$ is a *nominal* quantity, being measured in dollar, while
```
dollar -> dollars

jstac commented on lectures/unpleasant.md:
```diff
@@ -152,15 +154,15 @@ running monetary and fiscal policies.
 Here, by **fiscal policy** we mean the collection of actions that determine a sequence of net-of-interest government deficits $\{g_t\}_{t=0}^\infty$ that must be financed by issuing to the public  either money or interest bearing bonds.
 
-By **monetary policy** or **debt-management polcy**, we  mean the collection of actions that determine how the government divides its  portolio of debts to the public  between interest-bearing parts (government bonds) and non-interest-bearing parts (money).
+By **monetary policy** or *debt-management policy*, we  mean the collection of actions that determine how the government divides its  portolio of debts to the public  between interest-bearing parts (government bonds) and non-interest-bearing parts (money).
```
I think **debt-management policy** is OK here, since it's a definition.

mmcky replied:
thanks @jstac reading your comments as well to learn what to look out for. Thanks for this reminder re: definitions. 

https://manual.quantecon.org/styleguide/writing.html#emphasis-and-definitions

jstac commented on lectures/unpleasant.md:
````diff
@@ -501,11 +494,13 @@ mystnb:
 ```
 
-Figure  {numref}`fig:unpl1` summarizes outcomes of  two experiments that convey   messages of 
+{numref}`fig:unpl1` summarizes outcomes of  two experiments that convey   messages of 
````
please cut "Sargent and Wallace's **unpleasant monetarist arithmetic**" from this sentence.

jstac commented:
Many thanks @SylviaZhaooo and @mmcky 

Just some small changes requested.

SylviaZhaooo committed:
Update unpleasant.md

SylviaZhaooo committed:
Update unpleasant.md

SylviaZhaooo commented:
> Many thanks @SylviaZhaooo and @mmcky
> 
> Just some small changes requested.

I made the changes as requested, and the conversations have been resolved.

jstac commented:
Many thanks @SylviaZhaooo , much appreciated!  merging.
